### DISCUSSION TITLE
GitHub Student Developer Pack  Stuck on "Processing Information" Screen

### DISCUSSION THREAD

**Discussion by emirbilgili** [community/community#88719] [GitHub Education] [answered]
### Hi everyone,

Hello all :),

I have been trying for the past week to sign-up for GitHub Student Developer Pack so I can have access to the Co-Pilot AI in visual studio for a course I am taking (I go to UF).  

When do the required steps to sign-up/become eligible for it using my UF ID, it gets stuck on the "processing information" button and does not move forward as seen in the attachment. I have tried it a dozen times, on various computers, browsers and devices, all of them stay stuck here. I even left it running over night on this screen and nothing happened.

Any ideas on how to resolve this issue?

Here it what it looks like: 

![image](https://github.com/community/community/assets/151589878/d914a379-68c2-4f0c-9bd3-b3f27be5fd7b)
![Screenshot github education](https://github.com/community/community/assets/151589878/159af383-4548-487f-b061-5ae0fdf6ceaa)

Thank you

**Comment by AlessioBugetti** [comment]
Hi @emirbilgili , have you cleared your browser’s cache and cookies? Give that a shot and then try to sign up again. It’s a simple step, but it can often solve these kinds of issues. If that doesn’t work, try opening an incognito window in your browser and attempt to sign up through that. Another thing to consider is that browser extensions can sometimes interfere with website functions. If you have any extensions active, disable them temporarily and see if that makes a difference.

**Reply by emirbilgili** [reply]
Hello! Yes, I have tried that. Have used 3 different browsers, cleared cookies and on private browsing mode. I have even tried it on mobile and two other computers as well. Not sure what is up.

**Reply by AlessioBugetti** [reply]
At this point, I recommend reaching out to GitHub support.

**Reply by emirbilgili** [reply]
How do i do that? is there an email?

**Reply by AlessioBugetti** [reply]
You can use the [GitHub Support](https://support.github.com/) portal

**Reply by zen-zap** [reply]
I'm also having the same problem

**Reply by jab0314** [reply]
I'm having the same problem

**Reply by AnZhi2019** [reply]
I'm also having the same problem

**Reply by zen-zap** [reply]
Someone on the portal suggested taking the picture on your PC webcam... although the picture looks blurred, it was processed. I did that last night  so I still don't know If my request will get approved. It's worth a try

**Reply by KRAA5H** [reply]
Same problem and your solution is not working @AlessioBugetti

**Reply by emirbilgili** [reply]
That did not work either

**Reply by TallGibbs** [reply]
I'm having the same issue and I cannot use my computer webcam because my student ID does not have a date.  I tried to take a photo of my registration status webpage using my phone and uploading that JPG file.

**Comment by WoollySwain** [comment]
Been toiling with this for a while now but using a letter instead of a photo, I just cropped my letter to show the bare minimum and it finally processed. Try a different picture of your ID and crop it.

**Comment by KRAA5H** [comment]
Seems to be answered here. When I get home I will try. 
https://github.com/orgs/community/discussions/88401#discussion-6081413
https://github.com/orgs/community/discussions/86102#discussion-6053932
https://github.com/orgs/community/discussions/84817#discussion-6036465
https://github.com/orgs/community/discussions/86597#discussion-6061023
https://github.com/orgs/community/discussions/88446#discussion-6083620

**Reply by emirbilgili** [reply]
I tried many different browsers and computers also. I will try this again though

**Reply by KRAA5H** [reply]
I think there is a file size limit and an image size limit. GitHub does not say - just keeps it processing forever.

**Comment by KRAA5H** [comment, marked as answer]
The solution is to use the computer webcam. I did it and I processed taking me to this page. 
![image](https://github.com/community/community/assets/56270889/0e21bb62-60c1-4743-86c3-bcd73bbfca11)
Hope this helps @emirbilgili

**Reply by emirbilgili** [reply]
THIS WORKED FOR ME

**Comment by K-2LU** [comment]
Make sure the image is of the correct format(JPEG/JPG). I was trying with a PNG for hours as I overlooked that part. Finally noticed it and changed the image format and it finally processed. Haven't got the reply email yet, however wish me luck.
Hope this helps.

**Reply by TallGibbs** [reply]
I actually got it to work by shrinking the file size down.  Even though github says it can accept up to a certain size file, that does not seem to be true.  So, I just reduced the file size to only 25kb and then it processed immediately.

**Reply by K-2LU** [reply]
In my case image size wasn't the issue. But I'm glad it worked for you.

**Comment by zubairjammu786** [comment]
Doesn't work. there should be an upload button to upload files from pc directly. Fraustrated feature by Github
